static/app/components/replays/replayController.tsx

```diff
@@ -0,0 +1,236 @@
+              aria-label={t('Jump back 10 seconds')}
+            />
+            <Button
+              data-test-id="replay-play-pause"
+              size="xsmall"
+              title={isPlaying ? t('Pause the Replay') : t('Play the Replay')}
+              icon={isPlaying ? <IconPause size="sm" /> : <IconPlay size="sm" />}
+              onClick={() => togglePlayPause(!isPlaying)}
+              aria-label={t('Toggle real-time updates')}
+            />
+            <Button
+              data-test-id="replay-forward-10s"
+              size="xsmall"
+              title={t('Go forwards 10 seconds')}
+              icon={<IconRefresh color="gray500" size="sm" />}
+              onClick={() => setCurrentTime(currentTime + 10 * SECOND)}
+              aria-label={t('Jump forward 10 seconds')}
+            />
+          </ButtonBar>
+          <span>
+            {formatTime(currentTime)} / {duration ? formatTime(duration) : '??:??'}
+          </span>
+        </FlexLayout>
+        <FlexLayout direction="row">
+          <RightLeftBooleanField
+            data-test-id="replay-skip-inactive"
+            name="skip-inactive"
+            label={t('Skip to events')}
+            onChange={() => toggleSkipInactive(!isSkippingInactive)}
+            inline={false}
+            stacked
+            hideControlState
+            value={isSkippingInactive}
+          />
+          <ButtonBar active={String(speed)} merged>
+            {speedOptions.map(opt => (
+              <Button
+                key={opt}
+                size="xsmall"
+                barId={String(opt)}
+                onClick={() => onChangeSpeed(opt)}
+                title={t('Set playback speed to %s', `${opt}x`)}
+              >
+                {opt}x
+              </Button>
+            ))}
+          </ButtonBar>
+
+          <Button
+            data-test-id="replay-fullscreen"
+            size="xsmall"
+            title={t('View the Replay in full screen')}
+            icon={<IconResize size="sm" />}
+            onClick={() => onFullscreen()}
+            aria-label={t('View in full screen')}
+          />
+        </FlexLayout>
+      </ButtonLayout>
+    </FlexLayout>
+  );
+};
+
+type FlexDirectionValues =
+  | 'row'
+  | 'row-reverse'
+  | 'column'
+  | 'column-reverse'
+  | 'inherit'
+  | 'initial'
+  | 'revert'
+  | 'revert-layer'
+  | 'unset';
+
+type AlignItemsValues =
+  | 'normal'
+  | 'stretch'
+  | 'center'
+  | 'start'
+  | 'end'
+  | 'flex-start'
+  | 'flex-end'
+  | 'baseline'
+  | 'first baseline'
+  | 'last baseline'
+  | 'safe center'
+  | 'unsafe center'
+  | 'inherit'
+  | 'initial'
+  | 'revert'
+  | 'revert-layer'
+  | 'unset';
+
+const FlexLayout = styled('div')<{
+  alignItems?: AlignItemsValues;
+  direction?: FlexDirectionValues;
+  gap?: Parameters<typeof space>[0];
+}>`
+  display: flex;
+  flex-direction: ${p => p.direction ?? 'initial'};
+  gap: ${p => space(p.gap ?? 1)};
```

Comment:
We've been avoiding using `gap` for flexbox (opting only for grid) as we still have a small amt of users that use an unsupported browser https://caniuse.com/flexbox-gap

Reply:
ahh, i can do the css another way. This'll also impact the comment above regarding `FlexLayout` so i'll see how the whole area shakes out